packages/webapp/pages/onboarding.tsx

```diff
@@ -185,11 +202,21 @@ export function OnboardPage(): ReactElement {
     shouldEvaluate: shouldEnrollOnboardingStep && isSafariOnIOS(),
   });
 
+  const { isCurrentPWA, isAvailable: canUserInstallDesktop } = useInstallPWA();
+  const { value: installDesktopExperiment } = useConditionalFeature({
+    feature: featureOnboardingDesktopPWA,
+    shouldEvaluate:
+      shouldEnrollOnboardingStep &&
+      shouldShowExtensionOnboarding &&
```

Comment:
> Basically users are the same as onboarding_extension  + have to skip that step

I think we are missing the part where the user skipped the installation. This means, whether the user skipped or not, we would enroll them right away. We might have to opt to use the `feature.getFeatureValue` function and execute it within the function where the user skipped extension installation.

Reply:
Do we have some similar condition somewhere in codebase? Using `feature.getFeatureValue` will enroll the user only at that point? Just double checking avoiding potential issues @sshanzel